microcosm_pubsub/registry.py

```diff
@@ -115,6 +115,21 @@ def compute_bound_handlers(self, bindings):
             if handler in bound_components or handler in bound_component_types
         }
 
+        media_type_handlers = {}
```

Comment:
this is after a `return` statement

Reply:
worth adding a unittest?

Reply:
Probably misunderstanding, not sure what you mean. This is before the `return`.